enterprise/backend/src/metabase_enterprise/serialization/cmd.clj

```diff
@@ -203,15 +203,14 @@
 
 (defn v2-dump
   "Exports Metabase app data to directory at path"
-  [path {:keys [user-email collection-ids] :as opts}]
+  [path {:keys [collection-ids] :as opts}]
   (log/info (trs "Exporting Metabase to {0}" path) (u/emoji "🏭 🚛💨"))
   (mdb/setup-db!)
   (check-premium-token!)
   (t2/select User) ;; TODO -- why??? [editor's note: this comment originally from Cam]
```

Comment:
Tim's note: why???

Reply:
(Not removing it since that's certainly a Chesterton's Fence if I ever saw one…)